gitops/kcp/gateway/haproxy-ns.yaml

```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: gateway
```

Comment:
The other manifests reference `haproxy` instead of `gateway`. Typo?

Reply:
Not a typo, actually the NS is "gateway" and inside the ns the deployment, service, ingress are named "haproxy" since they are specific to haproxy.